docs-v2/mint.json

```diff
@@ -177,6 +177,7 @@
                 "integrations/all/google-sheet",
                 "integrations/all/gorgias",
                 "integrations/all/greenhouse",
+		"integrations/all/gumroad",
                 "integrations/all/gusto",
                 "integrations/all/hubspot",
```

Comment:
It looks like `hubspot` and `healthgorilla` are in the wrong order. Do you mind fixing this plz 🙏